stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -15,21 +15,151 @@ import _Concurrency
 
 // ==== Distributed Actor -----------------------------------------------------
 
+
+
 /// Common protocol to which all distributed actors conform implicitly.
 ///
-/// It is not possible to conform to this protocol manually explicitly.
-/// Only a 'distributed actor' declaration or protocol with 'DistributedActor'
-/// requirement may conform to this protocol.
+/// The `DistributedActor` protocol generalizes over all distributed actor types.
+/// Distributed actor types implicitly conform to this protocol.
+///
+/// It is not possible to conform to this protocol manually by any other type
+/// than a `distributed actor`.
+///
+/// It is possible to require a type conform to the
+/// ``DistributedActor`` protocol by refining it with another protocol,
+/// or by using a generic constraint.
+///
+///
+/// ## The ActorSystem associated type
+/// Every distributed actor must declare what type of distributed actor system
+/// it is part of by implementing the ``ActorSystem`` associated type requirement.
+///
+/// This causes a number of other properties of the actor to be inferred:
```

Comment:
This section should probably list properties that are going to be synthesized.

Reply:
Good idea, will add!

Reply:
Added an extra section on properties above, thanks!